packages/typegpu/tests/examples/individual/oklab.test.ts

```diff
@@ -197,7 +197,7 @@ describe('oklab example', () => {
       }
 
       fn gamutClipAdaptiveL05_9(lab: vec3f) -> vec3f {
-        var alpha = 0.2;
+        var alpha = 0.2f;
```

Comment:
The accessor's type was not carrying over to this variable declaration before these changes 🎉